cypress-tests-data.sh

```diff
@@ -0,0 +1,20 @@
+
+# literature-conference relation
+docker-compose $files exec web inspirehep importer records -f data/records/literature/1787272.json -f data/records/conferences/1787117.json
+
+#author search
+docker-compose $files exec web inspirehep importer records -f data/records/authors/1004662.json -f data/records/authors/1060898.json -f data/records/authors/1013725.json -f data/records/authors/1078577.json -f data/records/authors/1064002.json -f data/records/authors/1306569.json --save
+
+#job search
+docker-compose $files exec web inspirehep importer records -f data/records/jobs/1811684.json -f data/records/jobs/1812904.json -f data/records/jobs/1813119.json -f data/records/jobs/1811836.json -f data/records/jobs/1812529.json
+
+#seminar search
+docker-compose $files exec web inspirehep importer records -f data/records/seminars/1811573.json -f data/records/seminars/1811750.json -f data/records/seminars/1811657.json -f data/records/seminars/1807692.json -f data/records/seminars/1807690.json
+
+#conference search
+docker-compose $files exec web inspirehep importer records -f data/records/conferences/1769332.json -f data/records/conferences/1794610.json -f data/records/conferences/1809034.json -f data/records/conferences/1776122.json -f data/records/conferences/1622944.json
```

Comment:
Please notice that every execution of `inspirehep` takes additional ~12s to load all libraries. So here We have additional ~60s (~70s instead of ~12s from only one call). Isn't it better to load all files in one `inspirehep importer records` call?

Reply:
yeah I have been thinking that as well, I could convert to one